lib/cmake/obs-websocket-apiConfig.cmake.in

```diff
@@ -0,0 +1,8 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+find_dependency(libobs REQUIRED)
```

Comment:
Is `libobs` the only dependency for the API header?

Reply:
Yes, `proc_handler_t` is part of `libobs`.